librz/core/cmd_descs/cmd_debug.yaml

```diff
@@ -312,6 +312,25 @@ commands:
         summary: Debug continue until
         cname: cmd_debug_continue_until
         type: RZ_CMD_DESC_TYPE_OLDINPUT
+  - name: do
+    summary: Debug (re)open commands
+    subcommands:
+      - name: dor
+        summary: Comma separated list of k=v rz-run profile options (e dbg.profile)
+        cname: cmd_debug_process_dor
+        type: RZ_CMD_DESC_TYPE_OLDINPUT
+      - name: doe
+        summary: Show rz-run startup profile
+        cname: cmd_debug_process_profile
+        args: []
```

Comment:
I'm not sure what this command is supposed to do...
It prints things like:
```
setenv=TMUX_PANE="%38"
setenv=SHLVL="3"
setenv=XDG_SEAT="seat0"
setenv=BYOBU_ACCENT="#75507B"
setenv=BYOBU_TIME="%H:%M:%S"
setenv=BYOBU_RUN_DIR="/dev/shm/byobu-rschiron-uH9CIkbK"
```
what is `setenv=`?

Maybe this makes more sense under `dor`?